app/test/index.html

```diff
@@ -15,7 +14,16 @@
         'qea-question-type-test.html',
         'qea-questions-list-test.html',
         'qea-redux-store-test.html',
-        'qea-selector-highlighter-test.html'
+        'qea-selector-highlighter-test.html',
+        'qea-items-list-test.html',
+        'qea-user-responce-test.html',
+        'qea-toolbar-test.html',
+        'qea-main-app-test.html',
+        'qea-canvas-test.html',
+        'qea-editors-behavior-test.html',
+        'asq-text-input-q-editor-test.html',
+        'asq-multi-choice-q-editor-test.html',
+        'asq-buckets-q-editor-test.html'
```

Comment:
Can you also add the shadow dom variant like so: https://github.com/PolymerElements/iron-icon/blob/master/test/index.html#L21